README.md

```diff
@@ -59,8 +59,8 @@ $uri = 'https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim'
 ###### Unix
 
 ```sh
-curl -fLo ~/.local/share/nvim/site/autoload/plug.vim --create-dirs \
-    https://raw.githubusercontent.com/junegunn/vim-plug/master/plug.vim
+sh -c 'curl -fLo "${XDG_DATA_HOME:-~/.local/share}"/nvim/site/autoload/plug.vim --create-dirs \
```

Comment:
`~` cannot be quoted.

```sh
$ ls "${FOOBAR:-~}"
ls: ~: No such file or directory
```

Reply:
You're right. I just pushed a fix that uses `$HOME`. Sorry about that!